renderer/routes/recent.svelte

```diff
@@ -0,0 +1,110 @@
+<script>
+  import { onMount } from 'svelte'
+  import { fade } from 'svelte/transition'
+  import { _ } from 'svelte-intl'
+  import { replace } from 'svelte-spa-router'
+  import { distinct, filter, map } from 'rxjs/operators'
+  import { Heading, Button, PlaylistTracksTable } from '../components'
+  import { load, changes, removals } from '../stores/playlists'
+  import { add } from '../stores/track-queue'
+  import {
+    invoke,
+    formatTimeLong,
+    fromServerChannel,
+    sumDurations
+  } from '../utils'
+
+  let playlist
+
+  $: playlistId = 151120
+  $: if (!playlist || playlist.id !== playlistId) {
+    loadPlaylist()
+  }
+  $: total = playlist && playlist.trackIds ? playlist.trackIds.length : 0
+
+  async function loadPlaylist() {
+    playlist = await load(playlistId)
+    if (!playlist) {
+      return replace('/recent')
+    }
+  }
+
+  onMount(() => {
+    const changeSub = changes
+      .pipe(
+        map(changed => changed.find(({ id }) => id === playlistId)),
+        filter(n => n),
+        distinct()
+      )
+      .subscribe(async changed => {
+        if (!changed.tracks) {
+          // then load tracks if not yet available
+          playlist = await load(playlist.id)
+        } else {
+          // update now in case of media change
+          playlist = changed
+        }
+      })
+
+    const trackSub = fromServerChannel(`track-changes`).subscribe(changed => {
+      if (
+        playlist &&
+        changed.some(({ id }) => playlist.trackIds.includes(id))
+      ) {
+        loadPlaylist()
+      }
+    })
+
+    return () => {
+      changeSub.unsubscribe()
+      trackSub.unsubscribe()
+    }
+  })
+</script>
+
+<style type="postcss">
+  section {
+    @apply flex flex-row text-left m-4 mt-0;
+  }
+
+  .meta {
+    @apply flex-grow text-xl text-right;
+  }
+
+  .tracks {
+    @apply m-4;
+  }
+</style>
+
+<div in:fade={{ duration: 200 }}>
+  {#if playlist}
+    <Heading
+      title={playlist.name}
+      image={'../images/bantersnaps-nE1K_qO2z38-unsplash.jpg'}
```

Comment:
Would you mind using this [image](https://unsplash.com/photos/Ab6nlmqcTYo) (1920x1280) please?
Position should be `center 45%` so we can see both the trompets and the hairy drumist :smile: .

If you find something that convey the idea of recent on [Unsplash](https://unsplash.com), I'm happy we use it.

Photo credits are [here](https://github.com/feugy/melodie/blob/d639082ccc000edaf92410940758a92c8e1a42eb/renderer/routes/settings.svelte#L28-L49)